homeassistant/components/knx/__init__.py

```diff
@@ -82,6 +82,7 @@
 SERVICE_KNX_SEND: Final = "send"
 SERVICE_KNX_ATTR_PAYLOAD: Final = "payload"
 SERVICE_KNX_ATTR_TYPE: Final = "type"
+SERVICE_KNX_RESPONSE: Final = "response"
```

Comment:
It would be nice to use the same schema as the other attributes: `SERVICE_KNX_ATTR_RESPONSE: Final = "response"`